scripts/install-fmt.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+
+# number of jobs to run
+NPROCS=1
+# Find available processor numbers based on different OS.
+OS="`uname -s`"
```

Comment:
```suggestion
OS="$(uname -s)"
```
[Use $(...) notation instead of legacy backticked `...`.](https://www.shellcheck.net/wiki/SC2006)